streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -174,7 +174,7 @@ public <S extends StateStore> S  getStateStore(final String name) {
                 " as the store is not connected to the processor. If you add stores manually via '.addStateStore()' " +
                 "make sure to connect the added store to the processor by providing the processor name to " +
                 "'.addStateStore()' or connect them via '.connectProcessorAndStateStores()'. " +
-                "DSL users need to provide the store name to '.process()', '.transform()', or '.transformValues()' " +
+                "DSL users need to provide the store name to '.process()', '.processValues()', or '.transformValues()' " +
```

Comment:
Are we leaving the `.transformValues()` because of the `KTable`?

Reply:
Yes.